### ISSUE TITLE
World made with flat mod doesn't save

### ISSUE THREAD

**Issue by JasiuWu2006** [ReMinecraftPE/mcpe#70]
Idk if anyone else experienced this bug but a world made using the flat mode doesn't save because if you open it without the mod enabled the world isn't flat anymore, what's even worse is that any blocks that were placed on the flat world don't show aswell.
What's funny is that my flat world with the shop saves correctly.
Also IDK if this is only because I use Winlator.

**Comment by iProgramMC** [comment]
No, it's an issue with the outdated flat world mod. Currently it doesn't use any of the level data code. The flat world mod was written in a time where I didn't actually have the current world generator, and I didn't exactly want to get rid of it.
